wavedroplet.js

```diff
@@ -167,7 +179,7 @@ function init(json) {
     });
 
     // TODO(katepek): Recalculate and redraw when resized
-    dim.height = (total_height - 3 * state.to_plot.length * dim.padding) / state.to_plot.length;
+    dim.height = (total_height - histHeight - (state.to_plot.length + 1) * dim.padding) / state.to_plot.length;
```

Comment:
Yes - I purposefully left the legend hanging off the bottom to prioritize giving space for the charts.  

My sense is the legend is just sort-of there for now.  It's useful, but a different solution might be more useful.  But, I'm not sure what that is yet and hope it will become more clear after I improve other parts of the viz.  That said, we should talk about what the need is that it's trying to fill, so I can think about what some of the options are for best filling that need and how to best use the space.